tests/saw/spec/handshake/handshake_io_lowlevel.saw

```diff
@@ -154,8 +154,13 @@ let setup_connection_common chosen_psk_null = do {
    cork_val <- crucible_fresh_var "corked" (llvm_int 2);
    crucible_ghost_value corked cork_val;
 
+   // We assume that the client_cert_auth_type_overridden flag in s2n_connection is set.
+   // If that flag is not set, auth_type / "cca_type" could be determined by the config
+   // or just set to a default value. Since we're primarily interested in the handshake
+   // here and the end result is the same, just consider the connection auth_type.
    cca_ov <- crucible_fresh_var "cca_ov" (llvm_int 8);
```

Comment:
Do we need to define cca_ov here at all if we're just assuming it's always set?

Reply:
We do, because defining it is how we're setting it. That's how the proof knows to model it as always set.